packages/flutter_tools/test/general.shard/build_system/targets/macos_test.dart

```diff
@@ -204,5 +180,3 @@ class FakeProcessResult implements ProcessResult {
   @override
   String stdout = '';
 }
```

Comment:
Could you add some tests for things that we know have broken before? E.g., track_widget_creation is set when it should be, `pod install` is run if the trigger conditions are true.

Reply:
I've removed pods from the assemble path for now, so the conditions for it haven't changed.

I added a test for track_widget_creation when I landed those changes in https://github.com/flutter/flutter/pull/37512/files